js/app.js

```diff
@@ -0,0 +1,80 @@
+// ==================== HTML ELEMENTS TO MANIPULATE =======================
+
+// Get user input for height
+var heightInput = document.getElementById("heightInput");
+// Get user input for char
+var charInput = document.getElementById("charToUse");
+// Button element
+var button = document.getElementById("inputButton");
+
+var numberOfChars = 1;
+var charString = "";
+
+// Pine tree object
+var pineTree = {
+  height: null,
+  charToUse: null
+};
+
+
+// ================================= FUNCTIONS =================================
+
+// Function to build tree
+function buildTree() {
```

Comment:
**Requirement not met:** Create a `tree` function that should build a pine tree out of a character in the Chrome dev tools console. It accepts a single object as an argument. . The object should have two key/value pairs.